sdk/src/wallet/operations/balance.rs

```diff
@@ -129,6 +132,9 @@ where
                             output_id.transaction_id().slot_index(),
                             slot_index,
                         )?;
+                        if let Ok(response) = self.client().get_output_mana_rewards(output_id, slot_index).await {
+                            balance.mana.rewards += response.rewards;
+                        }
```

Comment:
Should we so these only if the output is delegation/account? That could be a LOT of requests for nothing otherwise

Reply:
Hm actually at this point it can't be either of them, since we match on the output types in L63
Instead if should be moved in this closure `Output::Delegation(delegation) => {`

Reply:
Ah yeah, even better

Reply:
done